specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/datamigration.json

```diff
@@ -800,6 +1006,52 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{groupName}/providers/Microsoft.DataMigration/services/{serviceName}/serviceTasks/{taskName}/cancel": {
+      "post": {
+        "summary": "Cancel a service task",
+        "description": "The service tasks resource is a nested, proxy-only resource representing work performed by a DMS instance. This method cancels a service task if it's currently queued or running.",
+        "tags": [
+          "Service Task resource",
+          "Custom operation",
+          "POST"
+        ],
+        "operationId": "ServiceTasks_Cancel",
+        "x-ms-examples": {
+          "Tasks_Cancel": { "$ref": "./examples/ServiceTasks_Cancel.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/groupName"
+          },
+          {
+            "$ref": "#/parameters/serviceName"
+          },
+          {
+            "$ref": "#/parameters/taskName"
+          },
+          {
+            "$ref": "#/parameters/apiVersion"
+          }
+        ],
+        "responses": {
```

Comment:
Please check if this is a long running operation

Reply:
See above.